lib/widget/cards/reusable_card_base.dart

```diff
@@ -37,18 +37,54 @@ class ReusableCardBase extends StatelessWidget {
     this.description,
     this.color = Colors.white,
     this.routeTo,
-    this.height,
+    this.boxConstraints,
     this.elevation = 4,
     this.borderRadius = 8.00,
     this.padding = const EdgeInsets.all(12),
-    this.margin,
+    this.margin = const EdgeInsets.all(4.0),
     this.fallback,
     @required this.child,
     this.verticalAlignment = MainAxisAlignment.start,
   }) : assert(child.isNotEmpty, 'Child component must be declared');
 
   @override
   Widget build(BuildContext context) {
+    var col = Column(
+      mainAxisAlignment: verticalAlignment,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisSize: MainAxisSize.min,
+      children: child,
+    );
+    return Card(
+        margin: margin,
+        color: color,
+        elevation: elevation,
+        shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.all(Radius.circular(borderRadius))),
+        child: InkWell(
+          child: Ink(
+            padding: padding,
+            child: boxConstraints == null
+                ? col
+                : ConstrainedBox(
+                    constraints: boxConstraints,
+                    child: col,
+                  ),
+          ),
+          onTap: () {
+            if (routeTo != null) {
+              Navigator.pushNamed(context, routeTo);
+            } else {
+              if (fallback != null) {
+                fallback();
+              }
```

Comment:
do we want to silently fail here? I think at least print a warning or maybe throw an Exception ?

Reply:
We need to cater for cards that have no click action, for example `ppe_step_card.dart`.
But I'll make a quick change to mean `onTap` is null if there is no action, meaning there will be no touch feedback either, which is desirable.